Week-2/HTML-ADVANCED/RESTUARANT-REVIEWS/mainStreetCafe.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+<header>
+    <h1>Restaurant Reviews</h1>
+    <nav>
+        <a href="">Home</a>
+        <a href="">Add Review</a>
+        <a href="">About</a>
+    </nav>
+</header>
+<main>
+    <section>
+        <h2>Main Street Cafe</h2>
+        <img src= "" alt= "Main street Cafe"><br>
+        <span class="fa fa-star checked"></span>
+        <span class="fa fa-star checked"></span>
+        <span class="fa fa-star checked"></span>
+        <span class="fa fa-star checked"></span>
+        <span class="fa fa-star-half-full checked"></span>
+    </section>
+    <section>
+        <h3>Overview</h3>
+        <p>Cuisine: American,Grill, Gluten Free option</p>
+        <p>Price: $- $$</p>
+        <p>Featuring a wide variety of foods</p>
+    </section>
+
+    <section>
+        <h3>Reviews</h3>
```

Comment:
Star ratings can be made more accessible by using off-screen text or `aria-label`s to convey the meaning of the stars to screen reader users.

```
<span class="fa fa-star checked" aria-label="4 stars"></span>
<span class="fa fa-star-half-full checked" aria-label="and a half star"></span>

```